src/plugins/inputs/ignore.js

```diff
@@ -10,6 +10,7 @@ exports.input = function (network, chan, cmd, args) {
 	const client = this;
 	let target;
 	let hostmask;
+	let create_new_ignored_window = false;
```

Comment:
You don't really need this bool.
The structure that you propose is rather strange, prior to your changes the case statement was the last thing that happened, that should stay this way.